### PR TITLE
Add clarifications from the TAG Review

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -134,7 +134,73 @@ The Realms API does __not__ introduce a new evaluation mechanism. The code evalu
 
 JavaScript modules are associated with a global object and set of built-ins. Realms contain their own separate module graph which runs in the context of that Realm, so that a full JavaScript development experience is available.
 
-## <a name='Clarifications'></a>Clarifications
+## Clarifications from the TAG Review
+
+This proposal has been submitted for a [TAG Review](https://github.com/w3ctag/design-reviews/issues/542) and the following subtopics represent some of the clarifications [we presented](https://github.com/w3ctag/design-reviews/issues/542#issuecomment-698615062).
+
+### So do Realms only have the ECMAScript APIs available?
```
Comment (caridy) [inline explainer.md:141]: maybe moving this to a FAQs section at the end of the explainer, or something like that, is better, these are generic enough to not need to mention the tag review.

Comment (leobalter) [reply explainer.md]: Done

File: explainer.md [hunk 2]
```diff
@@ -556,3 +559,67 @@ document.body.removeChild(iframe);
...
+
+One thing to understand here is that Realms are generally intended to be a sort of metaprogramming construct, which would be used by frameworks and libraries to build emulated JS environments for developers. I understand the feedback that this concept may be difficult for JS developers to understand; probably an introduction in the explainer to show how múltiple globals in JS already work would help make this document more accessible. Either way, it is an underlying primitive in the platform.
+
+### Realm MVP or max/min
+
+> Maybe for consistency sake it would make sense to have an accessor to expose it as a realm, thought currently the only thing exposed is `globalThis` and `import` - but we assume that could be extended in the future.
+
+The initial Realms proposal had more content and more ways to access things. We tried to build a MVP and hope we can explore expansions of the API in the future.
+
+### How the Compartments proposal relates to Realms
```
Comment (caridy) [inline explainer.md:621]: just remove this section, no point to have it here.

**Review by caridy — APPROVED** [review]
LGTM, just one note about unnecessary section that should be removed.

**Pull request merged**